pkg/connector/capabilities.go

```diff
@@ -100,7 +115,7 @@ var fileCaps = event.FileFeatureMap{
 			"video/mp4": event.CapLevelFullySupported,
 		},
 		Caption:          event.CapLevelFullySupported,
-		MaxCaptionLength: MaxCaptionLength,
+		MaxCaptionLength: MaxTextLength,
 		MaxSize:          MaxFileSize,
 	},
 	event.CapMsgSticker: {
```

Comment:
The bridge doesn't seem to support stickers, so this entry should be removed entirely